src/godot-tools.ts

```diff
@@ -20,6 +20,10 @@ export class GodotTools {
 		this.client = new GDScriptLanguageClient(p_context);
 		this.client.watch_status(this.on_client_status_changed.bind(this));
 		this.connection_status = vscode.window.createStatusBarItem(vscode.StatusBarAlignment.Right);
+
+		setInterval(() => {
+			this.retry_callback();
+		}, get_configuration("reconnect_cooldown", 1000));
```

Comment:
The default was changed to 3000, so change it here as well:

```suggestion
		}, get_configuration("reconnect_cooldown", 3000));
```